### DISCUSSION TITLE
Issue section getting assigned necessary!

### DISCUSSION THREAD

**Discussion by amitesh-183** [community/community#51574] [Repositories] [answered]
### issue assignment

General

### my douts

Hey, there I have recently started contrubting to open source project and contributed to some projects and seen in issues section people requesting the issue to be assign to them. And my question is - is it necessary to get assigned because till now I have directly forked and started contributing without getting assigned please tell me!

**Comment by airtower-luna** [comment, marked as answer]
Assigning an issue to someone basically says that a person is working on the issue, or supposed to do it in the future.

There's no *technical* requirement to get assigned before you contribute. There could be an *organizational* requirement (as in, it's part of how people want to run their projects). You should always take a look at the README or CONTRIBUTING files (or other documentation) of a project and see what it says there about how to contribute. Most projects won't require assignments for outside contributors, it's much more likely to be used to split work between core developers (especially if it's part of their job).

**Reply by amitesh-183** [reply]
Thanks, It's been greatly helpful

**Reply by airtower-luna** [reply]
Glad I could help! Please don't forget to mark the answer. :slightly_smiling_face:
